src/Modules/ModulesGraded.jl

```diff
@@ -2052,6 +2084,8 @@ end
 
 elem_type(::Type{FreeMod_dec{T}}) where {T} = FreeModElem_dec{T}
 parent_type(::Type{FreeModElem_dec{T}}) where {T} = FreeMod_dec{T}
+elem_type(::FreeMod_dec{T}) where {T} = FreeModElem_dec{T}
+parent_type(::FreeModElem_dec{T}) where {T} = FreeMod_dec{T}
```

Comment:
Why are those methods necessary? https://github.com/oscar-system/Oscar.jl/pull/3068 went to great lengths to remove these.